Makefile

```diff
@@ -193,8 +193,14 @@ v8:
 	$(MAKE) -C deps/v8 $(V8_ARCH).$(BUILDTYPE_LOWER) $(V8_BUILD_OPTIONS)
 
 test: all
+	$(MAKE) test-build
+	$(MAKE) test-run
+
+test-build:
 	$(MAKE) build-addons
 	$(MAKE) build-addons-napi
+
+test-run:
 	$(MAKE) cctest
```

Comment:
Shouldn't this move to `build-test` as it triggers `all` as well:
https://github.com/nodejs/node/blob/8f7e2684e97a6dc37c1b6598524e5f37c35d4cb3/Makefile#L182-L183